### PR TITLE
eii-evam:Readme updated

### PR DESCRIPTION
**Pull request by kamalraj21intel**
Changes:
Added note for EII name
Renamed instances of EII to OEI
Minor fixes for markdown and grammar errors

Signed-off-by: kamal <kamal.raj@intel.com>

### CODE REVIEW

**Comment by kamalraj21intel** [comment]
@iamvishwas Please review and provide feedback. I am unable to tag Vinod here, I'll ping him the link to this MR in case he wants to review this.

File: README.md [hunk 1]
```diff
@@ -1,53 +1,71 @@
-# Edge Video Analytics Microservice
```
Comment (iamvishwas) [inline README.md:1]: Please update the eii/README in this repo as well @kamalraj21intel

**Comment by iamvishwas** [comment]
Please review @yogeshmpandey @mdiqbal88

File: README.md (hunk 1 above)
Comment (kamalraj21intel) [reply README.md:1]: Hi, @iamvishwas,
I've updated the Readme. If any other changes are required that please provide the input for the update.

Comment (iamvishwas) [reply README.md:1]: Hi @kamalraj21intel,

This README is to be updated as well https://github.com/intel/edge-video-analytics-microservice/blob/master/eii/README.md

Comment (kamalraj21intel) [reply README.md:1]: Thank @iamvishwas, I've updated the readme, please review and provide your feedback.

File: README.md [hunk 2]
```diff
@@ -1,53 +1,71 @@
...
+		- [Run the base image](#run-the-base-image)
+		- [Run EVAM in OEI mode](#run-evam-in-oei-mode)
 
+## Edge Video Analytics Microservice
 
-## Building the image 
+This repository contains the source code for Edge Video Analytics Microservice (EVAM) used for the [Video Analytics Use Case](https://www.intel.com/content/www/us/en/developer/articles/technical/video-analytics-service.html). For information on how to build the use case, refer to the [Get Started](https://www.intel.com/content/www/us/en/developer/articles/technical/video-analytics-service.html#inpage-nav-3) guide.
 
-- To build the base image, run the following command
+>**Note:** In this document, you will find labels of ‘Edge Insights for Industrial (EII)’ for filenames, paths, code snippets, and so on. Consider the references of EII as Open Edge Insights (OEI). This is due to the product name change of EII as OEI.
```
[the remark this answers is not in the document]
Comment (iamvishwas) [reply README.md:13]: One minor comment, approved.

Please review this PR since it changes some content in main README as well @yogeshmpandey @mdiqbal88

**Review by yogeshmpandey — APPROVED** [review]
Approved

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kamalraj21intel) [reply README.md]: @iamvishwas Review comments are incorporated.

**Pull request merged**